test.sh.in

```diff
@@ -59,18 +69,41 @@ dlTftpFiles() {
 
 prepareDeps() {
   rm -rf ${WORKSPACE}/build-deps
-  mkdir -p ${WORKSPACE}/build-deps/${REPO_NAME} 
+  if [ -z "${MULTI}" ]; then
+    mkdir -p ${WORKSPACE}/build-deps/${REPO_NAME} 
+  else
+    mkdir -p ${WORKSPACE}/build-deps
+    mkdir -p ${WORKSPACE}/build
+  fi
   for i in ${REPOS[@]}; do
      cd ${WORKSPACE}/build-deps
-     if [ "${i}" != "${REPO_NAME}" ]; then
-         git clone https://github.com/RackHD/${i}.git
+     if [[ "${REPO_NAME}" != *"${i}"* ]]; then
+         git clone https://github.com/changev/${i}.git
          cd ${i}
          git checkout ${GIT_REFSPEC}
          npm install --production
      fi
   done
   dlTftpFiles
   dlHttpFiles
+  apiPackageModify
+}
+
+apiPackageModify() {
+  if [[ "${REPO_NAME}" == *"on-http"* ]] && [[ "${REPO_NAME}" == *"rackhd"* ]]; then
+    pushd ${WORKSPACE}/on-http/extra
+    sed -i "s/.*git symbolic-ref.*/ continue/g" make-deb.sh
+    sed -i "/build-package.bash/d" make-deb.sh
+    sed -i "/mkdir/d" make-deb.sh
+    bash make-deb.sh
+    popd
+    for package in $(echo "on-http-api1.1 on-http-api2-0 on_http_redfish_1_0"); do
```

Comment:
A new feature is added today and some tests are still needed. When every thing done I'll change it to RackHD/xxx repo.